mocks/mocks.go

```diff
@@ -343,7 +343,19 @@ func (*Publisher) SubmitToCT(_ context.Context, der []byte) error {
 // calls (which are most of what we use).
 type Statter struct {
 	statsd.NoopClient
-	Counters map[string]int64
+	Counters           map[string]int64
+	LastTimingDuration LastStatsdCommand
```

Comment:
Why do you only want to store LastTimingDuration rather than following the example of Counters and having a `map[string][]int64`? That would be more flexible for cases where multiple TimingDurations can be emitted.